oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticIndexStatistics.java

```diff
@@ -205,13 +216,15 @@ private StatsResponse stats(StatsRequestDescriptor crd) throws IOException {
             }
             // Assuming a single index matches crd.index
             IndicesRecord record = records.get(0);
-            String size = record.storeSize();
+            String storeSize = record.storeSize();
+            String primaryStoreSize = record.storeSize();
```

Comment:
Should this be `record.priStoreSize()` instead ?

Reply:
good catch. Thanks!